Libraries/ActionSheetIOS/RCTActionSheetManager.m

```diff
@@ -149,33 +149,23 @@ - (void)presentViewController:(UIViewController *)alertController
     item.target = self;
     [menu addItem:item];
   }
-  
-  NSPoint origin = NSZeroPoint;
-  NSEvent *event = nil;
+
   RCTPlatformView *view = nil;
   if (anchorViewTag) {
     view = [self.bridge.uiManager viewForReactTag:anchorViewTag];
-    event = [view.window currentEvent];
   }
-  NSView *superview = [view superview];
-  if (event && view) {
-    // On a macOS trackpad, soft taps are received as sysDefined event types. SysDefined event locations are relative to the screen so we have to convert those separately.
-    NSPoint eventLocationRelativeToWindow = NSZeroPoint;
-    CGPoint eventLocationInWindow = [event locationInWindow];
-    if ([event type] == NSEventTypeSystemDefined) { // light tap event relative to screen
-      eventLocationRelativeToWindow = [[view window] convertRectFromScreen:NSMakeRect(eventLocationInWindow.x, eventLocationInWindow.y, 0, 0)].origin;
-    } else { // full click events are relative to the window
-      eventLocationRelativeToWindow = eventLocationInWindow;
-    }
-    origin = [view convertPoint:eventLocationRelativeToWindow fromView:nil];
-  } else if (view) {
-    CGRect superviewFrame = [superview frame];
-    origin = NSMakePoint(NSMidX(superviewFrame), NSMidY(superviewFrame));
+  CGPoint location = CGPointZero;
+  NSView *inView = nil;
+  if (view != nil) {
+    // Display under the anchorview
+    inView = [view superview];
+    CGRect frame = [view frame];
```

Comment:
May be easier to just use bounds and present in view? That way the coordinates can be fully dependent on the view itself (either 0 or size.height/width)